crates/subspace-networking/src/node_runner.rs

```diff
@@ -113,8 +116,10 @@ where
     established_connections: HashMap<(PeerId, ConnectedPoint), usize>,
     /// Defines protocol version for the network peers. Affects network partition.
     protocol_version: String,
-    /// Defines whether we maintain a persistent connection.
-    connection_decision_handler: ConnectionDecisionHandler,
+    /// Defines whether we maintain a persistent connection for common peers.
+    general_connection_decision_handler: ConnectionDecisionHandler,
```

Comment:
`ConnectionDecisionHandler` name is not great, it is not clear what kinds of decisions it is making :confused: 

Maybe `ConnectedPeersHandler` or `KeepAliveHandler`?